servlet/pom.xml

```diff
@@ -19,6 +19,7 @@
         <module>pushBuilder</module>
         <module>servletContextListener</module>
         <module>servletRequestListener</module>
+        <module>theLearningServlet</module>
```

Comment:
Please rename theLearningServlet to explainingHttpServlet (as that is what you are explaining here)